style/style.css

```diff
@@ -0,0 +1,152 @@
+* {
+  padding: 0;
+  margin: 0;
+  box-sizing: border-box;
+}
+
+body {
+  width: 100%;
+  background: url("https://web.archive.org/web/20140228114747im_/http://images.apple.com/v/home/as/images/your_verse_hero.jpg") 50% 100% no-repeat;
+  height: 68vh;
+}
+
+footer a,
+footer ul a {
+  text-decoration: none;
+  color: #09c;
+}
+
+header ul a {
+  text-decoration: none;
+}
+
+.nav-menu ul {
+  display: flex;
+  height: 2.5rem;
+  width: 80vw;
+  margin: 5.2rem auto 0 auto;
+  border-radius: 0.3rem;
+  text-align: center;
+  background-image: linear-gradient(#808080 50%, #5e5e5e 30%, #808080 90%);
+}
+
+ul li {
+  list-style-type: none;
+}
+
+.nav-menu ul li {
+  border-right: 1px solid #666;
+  width: 7.49rem;
+  padding-top: 0.54rem;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 1.2rem;
+}
+
+nav .footer-nav li {
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 0.7rem;
+  margin-right: 0.4rem;
+  align-content: center;
+}
+
+.nav-menu ul li:hover {
+  background-color: #5e5e5e;
+}
+
+.nav-menu ul .last-item:hover {
+  background-color: transparent;
+}
+
+.nav-menu ul li:last-child {
+  border-right: none;
+}
+
+.page-highlight a {
+  color: #fff;
+  text-decoration: none;
+  font-size: 1.3rem;
+}
+
+.nav-menu ul a {
+  color: #fff;
+}
+
+input[type=text] {
+  width: 6rem;
+  height: 1.5rem;
+  border-radius: 0.8rem;
+  border: 2px solid #5e5e5e;
+  background: inherit;
+  margin-left: 0.4rem;
+}
+
+.fa-search {
+  position: relative;
+  right: 5.5rem;
+  color: #fff;
+  font-size: 0.8rem;
+}
+
+.call-to-action {
+  text-align: center;
+  color: #fff;
+  font-family: "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+  background: url("https://web.archive.org/web/20140228160945im_/http://images.apple.com/home/images/your_verse_title_large.png") 50% 10% no-repeat;
+}
+
+.page-highlight h2 {
+  padding-top: 8rem;
+  padding-bottom: 1.4rem;
+  font-size: 1.5rem;
+  line-height: 2rem;
+}
+
+.product-sample ul {
+  display: flex;
+  position: absolute;
+  top: 80vh;
+}
+
+nav ul img {
+  width: 1.1rem;
+  height: 0.9rem;
+  border-radius: 50%;
+}
+
+.product-sample ul img {
+  width: 100%;
+}
+
+.product-types {
+  display: none;
+}
+
+footer p {
+  font-size: 0.8rem;
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+footer {
+  padding: 1.5rem 7rem;
+  display: flex;
+  justify-content: space-between;
+  position: absolute;
+  margin-top: 27%;
+  width: 100%;
+}
```

Comment:
- [ ] Your footer is overlapping the four images section because it's positioned absolute. Please fix both and make sure they behave correctly.

- [ ] the rightmost elements should have a vertical divisor between each link, pĺease add it.